src/utils/components/AdditionalResources/AdditionalResources.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+
+import { V1Template } from '@kubevirt-ui/kubevirt-api/console';
+import VirtualMachineModel from '@kubevirt-ui/kubevirt-api/console/models/VirtualMachineModel';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import {
+  DescriptionListDescription,
+  DescriptionListGroup,
+  DescriptionListTerm,
+} from '@patternfly/react-core';
+
+type AdditionalResourcesProps = {
+  template: V1Template;
+};
+
+const AdditionalResources: React.FC<AdditionalResourcesProps> = ({ template }) => {
+  const { t } = useKubevirtTranslation();
+
+  if (!template?.objects || template?.objects.length === 1) return null;
+
+  return (
+    <DescriptionListGroup>
+      <DescriptionListTerm>{t('Additional resources')}</DescriptionListTerm>
+      <DescriptionListDescription>
+        <ul>
+          {template?.objects
+            ?.filter((object) => object?.kind && object.kind !== VirtualMachineModel.kind)
```

Comment:
VirtualMachineModel.kind is guaranteed as its hardcoded, so compare of object?.kind !== VirtualMachineModel.kind , is sufficient here i guess. :)